app/api/video_channel.py

```diff
@@ -18,7 +18,7 @@ def before_get(self, args, kwargs):
 
     methods = ['GET', 'POST']
     decorators = (api.has_permission('is_admin', methods="POST"),)
-    schema = VideoChannelSchemaPublic
+    schema = VideoChannelSchema
```

Comment:
Wrong. This will show API key to everyone. We already set proper schema according to admin in before_get